erizo/generateEclipseProject.sh

```diff
@@ -6,7 +6,7 @@ BIN_DIR="build"
 if [ -d $BIN_DIR ]; then
   cd $BIN_DIR
   # Set to Debug to be able to debug in Eclipse
-  cmake -G"Eclipse CDT4 - Unix Makefiles" -D ERIZO_BUILD_TYPE=debug ../src
+  cmake -G "Eclipse CDT4 - Unix Makefiles" -D ERIZO_BUILD_TYPE=debug ../build/debug
```

Comment:
I think this line might be wrong... can you try with this version of generateEclipseProject.sh I made by adding the Eclipse label please?

```
#!/usr/bin/env bash
set -e

SCRIPT=`pwd`/$0
FILENAME=`basename $SCRIPT`
PATHNAME=`dirname $SCRIPT`
BASE_BIN_DIR="build"


generateVersion() {
  echo "generating $1"
  BIN_DIR="$BASE_BIN_DIR/$1"
  if [ -d $BIN_DIR ]; then
    cd $BIN_DIR
  else
    mkdir -p $BIN_DIR
    cd $BIN_DIR
  fi
  cmake ../../src "-DERIZO_BUILD_TYPE=$1" -G"Eclipse CDT4 - Unix Makefiles"
  cd $PATHNAME
}


generateVersion debug
```